api/schema/src/main/resources/org/apache/isis/schema/chg/chg-2.0.xsd

```diff
@@ -24,11 +24,11 @@
            xmlns:com="http://isis.apache.org/schema/common"
 >
 
-    <xs:import namespace="http://isis.apache.org/schema/common" schemaLocation="../common/common-1.1.xsd"/>
+    <xs:import namespace="http://isis.apache.org/schema/common" schemaLocation="../common/common-2.0.xsd"/>
```

Comment:
the "default" attribute of these two elements:

<xs:element name="majorVersion" type="xs:string" minOccurs="0" maxOccurs="1" default="1">
<xs:element name="minorVersion" type="xs:string" minOccurs="0" maxOccurs="1" default="1">

should be changed in line version (ie to default="2" and default="0." respectively).

This applies to all of the updated .xsd files.